types/types_test.go

```diff
@@ -8,6 +8,12 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+var (
+	addr1 = Address{0x1}
+	addr2 = Address{0x2}
+	addr3 = Address{0x3}
+)
```

Comment:
Delete these and define them in tests, please avoid having global test variables

Reply:
Fixed 3db17e140e87d91c1fcbcf64d8df1ed0ef886965